docs/docs/refs-and-the-dom.md

```diff
@@ -73,7 +73,23 @@ class AutoFocusTextInput extends React.Component {
 }
 ```
 
-You may not use the `ref` attribute on functional components because they don't have instances. You can, however, use the `ref` attribute inside the functional component:
+**You may not use the `ref` attribute on functional components** because they don't have instances. In the below code the ref will always be called with `null`:
+
+```
+function StatelessComponent() {
```

Comment:
Let's call it `FunctionalComponent`. That's the important part, not whether it uses state or not.
Maybe even `MyFunctionalComponent` to make it clear `FunctionalComponent` is not some special name.